randovania/data/json_data/prime1.txt

```diff
@@ -2403,582 +2411,1261 @@ Ruins Entrance
+      Morph Ball Bomb and Space Jump Boots and Morph Ball
+
+> Pickup (Missile Expansion); Heals? False
+  * Pickup 18; Major Location? False
+  > Dock to East Atrium
+      Trivial
 
 ----------------
 Totem Access
 Asset id: 3934929011
 > Dock to Ruined Gallery; Heals? False
   * Normal Door to Ruined Gallery/Dock to Totem Access
+  > Dock to Hive Totem
+      Trivial
 
 > Dock to Hive Totem; Heals? False
   * Normal Door to Hive Totem/Dock to Totem Access
+  > Dock to Ruined Gallery
+      Trivial
 
 ----------------
 Map Station
 Asset id: 442183190
 > Dock to Ruined Gallery; Heals? False
-  * Normal Door to Ruined Gallery/Dock to Map Station
+  * Missile Blast Shield to Ruined Gallery/Dock to Map Station
 
 ----------------
 Sun Tower Access
 Asset id: 1103925484
 > Dock to Sunchamber; Heals? False
   * Normal Door to Sunchamber/Dock to Sun Tower Access
+  > Dock to Sun Tower
+      Trivial
 
 > Dock to Sun Tower; Heals? False
   * Normal Door to Sun Tower/Dock to Sun Tower Access
+  > Dock to Sunchamber
+      Trivial
 
 ----------------
 Hive Totem
 Asset id: 3358629366
 > Dock to Totem Access; Heals? False
   * Normal Door to Totem Access/Dock to Hive Totem
+  > Dock to Transport Access North
+      Trivial
 
 > Dock to Transport Access North; Heals? False
-  * Normal Door to Transport Access North/Dock to Hive Totem
+  * Missile Blast Shield to Transport Access North/Dock to Hive Totem
+  > Dock to Totem Access
+      Space Jump Boots and After Hive Totem one-time falling blocks
+  > Pickup (Missile Launcher)
+      Trivial
+  > Event - One-time falling blocks
+      After Hive Mecha
+
+> Pickup (Missile Launcher); Heals? False
+  * Pickup 19; Major Location? True
+  > Dock to Transport Access North
+      Trivial
+  > Event - Hive Mecha
+      Trivial
+
+> Event - Hive Mecha; Heals? False
+  * Event Hive Mecha
+
+> Event - One-time falling blocks; Heals? False
+  * Event Hive Totem one-time falling blocks
+  > Dock to Totem Access
+      Trivial
 
 ----------------
 Sunchamber
 Asset id: 2584347627
 > Dock to Sun Tower Access; Heals? False
   * Normal Door to Sun Tower Access/Dock to Sunchamber
+  > Dock to Sunchamber Access
+      After Chozo Ghosts (Sunchamber)
+  > Pickup (Varia Suit)
+      After Flaahgra
+  > Pickup (Artifact of Wild)
+      Space Jump Boots and After Chozo Ghosts (Sunchamber)
+  > Event - Chozo Ghosts
+      After Activate Sunchamber Chozo Ghosts layer
 
 > Dock to Sunchamber Access; Heals? False
   * Normal Door to Sunchamber Access/Dock to Sunchamber
+  > Dock to Sun Tower Access
+      After Chozo Ghosts (Sunchamber)
+  > Event - Flaahgra
+      Trivial
+
+> Pickup (Varia Suit); Heals? False
+  * Pickup 20; Major Location? True
+  > Dock to Sun Tower Access
+      Trivial
+
+> Pickup (Artifact of Wild); Heals? False
+  * Pickup 21; Major Location? True
+  > Dock to Sun Tower Access
+      Trivial
+
+> Event - Flaahgra; Heals? False
+  * Event Flaahgra
+  > Dock to Sun Tower Access
+      Trivial
```

Comment:
This node should connect to the varia pickup, with morph and bombs as the requirement.

Reply:
Done